waku/v2/waku_discv5.nim

```diff
@@ -89,6 +126,15 @@ proc new*(T: type WakuDiscoveryV5,
         )
         builder.withWakuCapabilities(flags)
         builder.withMultiaddrs(multiaddrs)
+
+        if relayShardsRes.isOk():
+          let op = relayShardsRes.value
+          if op.isSome():
+            let shards = relayShardsRes.value.get()
```

Comment:
```suggestion
            let shards = op.get()
```
perhaps?

Reply:
I might rewrite this it look funky but yes your right!